### DISCUSSION TITLE
Gradel buid failed

### DISCUSSION THREAD

**Discussion by prafulbbandre** [oppia/oppia-android#5091] [Q&A (Installation)] [answered]
### Operating System

windows 10

### Android Studio Version

flamingo

### RAM size

20gb

### Free disk space

380

### Which step are you stuck on?

build

### Error Log

Caused by: org.gradle.api.InvalidUserCodeException: Entry arguments.proto is a duplicate but no duplicate handling strategy has been set. Please refer to https://docs.gradle.org/7.2/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy for details.

Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final java.lang.String java.io.File.path accessible: module java.base does not "opens java.io" to unnamed module @245d4978

**Comment by prafulbbandre** [comment]
Hi @rt4914 @MohitGupta121 @adhiamboperes @BenHenning 
Please Help

**Comment by MohitGupta121** [comment]
@prafulbbandre Can you please make a clean and new folder and re-clone the project. Also are you doing the Step no.2 mentioned here in the [wiki](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android#install-oppia-android) ( bash scripts/setup.sh).

**Reply by prafulbbandre** [reply]
i had mention earlier that i am not having permission to copy paste the second point

**Reply by prafulbbandre** [reply]
i am trying on it

**Reply by MohitGupta121** [reply]
Why do you not have permission, it's your personal computer. Please use C-drive then to setup.

**Reply by prafulbbandre** [reply]
will try asap

**Reply by prafulbbandre** [reply]
@MohitGupta121 i had added it to the git folder still no change showing the same error.

**Reply by Abinash6000** [reply]
I had the same problem. So, it happened because I changed the graddle-wrapper.properties.
```
distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
```
I changed this to a different version. So, as @MohitGupta121 said, reclone. This could be because you've changed some files which shouldn't have been changed...

**Reply by MohitGupta121** [reply]
Why you changing gradle properties, please use the recommend Android studio version for now.

**Reply by prafulbbandre** [reply]
the update for it pops up frequently so i also upgraded it, I had started to reclone the project will update you soon.

**Reply by Abinash6000** [reply]
You're right. I'm just saying it was my mistake.

**Reply by prafulbbandre** [reply, marked as answer]
i down graded the jdk to 11 and the setup now runs.
Thanks @adhiamboperes @MohitGupta121
